app/ui/src/app/common/ui-patternfly/syndesis-form-control.scss

```diff
@@ -8,6 +8,13 @@ textarea:read-only {
   background-color: transparent;
   border: none;
 }
-.control-label > label {
-  margin-bottom: 0;
+.control-label {
+  &:hover {
```

Comment:
You can save the hover combinator here, since you’ll want to display a cursor pointer all the time.